chainer/gradient_check.py

```diff
@@ -370,7 +370,7 @@ class _CheckBackward(object):
 
     def __init__(
             self, func, x_data, y_grad, params, eps, atol, rtol, no_grads,
-            dtype, detect_nondifferentiable):
+            dtype, detect_nondifferentiable, params_to_func):
```

Comment:
Isn't there any better name than `params_to_func`?
I feel it's difficult to tell the meaning.

Reply:
Fixed.